scripts/data/common.py

```diff
@@ -0,0 +1,24 @@
+"""Common helpers for data."""
+from __future__ import annotations
+
+import sys
+from typing import Any
+
+import voluptuous as vol
+
+
+def expand_and_humanize_error(content: dict[str, Any], error: vol.Invalid|vol.MultipleInvalid) -> list[str] | str:
```

Comment:
`vol.MultipleInvalid` is a subclass of `vol.Invalid`:
```suggestion
def expand_and_humanize_error(content: dict[str, Any], error: vol.Invalid) -> list[str] | str:
```

Reply:
Right 👍